src/components/SuggestionItem/SuggestionItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useId } from 'react';
+import { bem } from '../../utils';
+import { Checkbox } from '../Checkbox';
+import styles from './SuggestionItem.scss';
+
+export interface Props {
```

Comment:
we had some discussions in the context of designing SelectedItemBadge. It will render a Dropdown where this (SelectedItem) and other similar components will be used as children. For that reason we need this component to support all the props that Dropdown children need to support. Can you please make sure they are there?